### PR TITLE
Add instructions to launch UWP from Visual Studio

### PR DESCRIPTION
**Pull request by vnen**
Also add a detail and correct typos in the export for UWP page.

### CODE REVIEW

File: reference/compiling_for_uwp.rst [hunk 1]
```diff
@@ -74,3 +74,57 @@ Zip according to the target/architecture of the template::
 Move those templates to the ``templates`` folder in Godot settings path. If
 you don't want to replacet the templates, you can set the "Custom Package"
 property in the export window.
+
+Running UWP apps with Visual Studio
+-----------------------------------
+
+If you want to debug the UWP port or simply run your apps without packaging
+and signing, you can deploy and launch them using Visual Studio. It might be
+the easiest way if your testing on a device such as a Windows Phone or an
```
Comment (akien-mga) [inline reference/compiling_for_uwp.rst]: s/your/you are/

File: reference/compiling_for_uwp.rst [hunk 2]
```diff
@@ -74,3 +74,57 @@ Zip according to the target/architecture of the template::
...
+Xbox One.
+
+Within the ANGLE source folder, open ``templates`` and double-click the
+``install.bat`` script file. This will install the Visual Studio project
+templates for ANGLE apps.
+
+If you have not built Godot, open the ``winrt/10/src/angle.sln`` solution
+from the ANGLE source and build it to Release/Win32 target. You may also need
+to build it for ARM if plan to run on a device. You can also use MSBuild if
```
Comment (akien-mga) [inline reference/compiling_for_uwp.rst]: if *you* plan

File: reference/compiling_for_uwp.rst (hunk 1 above)
Comment (vnen) [inline reference/compiling_for_uwp.rst]: Oops...

File: reference/compiling_for_uwp.rst [hunk 3]
```diff
@@ -74,3 +74,57 @@ Zip according to the target/architecture of the template::
...
+In the same "Binaries" filter, select "Add > Existing Item" and point to the
+Godot executable for UWP you have. In the "Properties" window, set "Content"
+to ``True`` so it's included in the project.
+
+Right-click the ``Package.appxmanifest`` file and select "Open With... > XML
+(Text) Editor". In the ``Package/Applications/Application`` element, replace
+the ``Executable`` attribute from ``$targetnametoken$.exe`` to
+``godot.winrt.exe`` (or whatever your Godot executable is called). Also change
+the ``EntryPoint`` attribute to ``GodotWinRT.App``. This will ensure that
+the Godot executable is correctly called when the app start.
```
Comment (akien-mga) [inline reference/compiling_for_uwp.rst]: s/start/starts/

**Pull request merged**